src/parquet/file/reader.cc

```diff
@@ -54,11 +54,11 @@ int64_t RowGroupReader::num_rows() const {
   return contents_->num_rows();
 }
 
-std::shared_ptr<ColumnReader> RowGroupReader::Column(int i) {
+std::shared_ptr<ColumnReader> RowGroupReader::Column(int i, int64_t chunk_size) {
```

Comment:
Per above having this parameter here feels like an unpleasant detail that should be handled higher up in the IO configuration for the file reader